libs/application/templates/children-residence-change/src/lib/messages/parentBConfirmation.ts

```diff
@@ -0,0 +1,41 @@
+import { defineMessage, defineMessages } from 'react-intl'
+
+// parentBConfirmation
+export const parentBConfirmation = {
+  general: defineMessages({
+    pageTitle: {
+      id: 'crc.application:section.parentBConfirmation.pageTitle',
+      defaultMessage: 'Umsókn um breytt lögheimili móttekin',
+      description: 'Parent B Confirmation page title',
+    },
+    description: {
+      id: 'crc.application:section.parentBConfirmation.description',
+      defaultMessage:
+        'Umsókn þín um breytt lögheimili hefur verið móttekin og send nú verið send áfram í vinnslu hjá sýslumanni.',
```

Comment:
I feel like this sentence is a little wonky?
```suggestion
        'Umsókn þín um breytt lögheimili hefur verið móttekin og send send áfram í vinnslu hjá sýslumanni.',
```